onmt/translate/Translator.lua

```diff
@@ -28,11 +28,15 @@ end
 
 function Translator:__init(args)
   self.opt = args
-  onmt.utils.Cuda.init(self.opt)
```

Comment:
I propose you update the server code - because we have a strange mix here. what do you think?